packages/entities/entities-plugins/src/plugins-endpoints.ts

```diff
@@ -20,8 +20,14 @@ export default {
   },
   form: {
     konnect: {
-      create: '/v2/control-planes/{controlPlaneId}/core-entities/plugins',
-      edit: '/v2/control-planes/{controlPlaneId}/core-entities/plugins/{id}',
+      create: {
+        all: '/v2/control-planes/{controlPlaneId}/core-entities/plugins',
+        forEntity: '/v2/control-planes/{controlPlaneId}/core-entities/{entityType}/{entityId}/plugins',
```

Comment:
Do we know if this endpoint exists? I feel like I might have tried this at one point and gotten a 404, but I might be misremembering

Reply:
I think I tested this endpoint with some `entityTypes`, but let me do a thorough test because I definitely don't want to break Konnect with this change.

Reply:
| create | update |
| -- | -- |
|When `entityType` is service: `/services/:serviceId/plugins`<img width="1164" alt="image" src="https://github.com/Kong/public-ui-components/assets/10095631/3e32b994-4e9a-4892-8579-19fff43fde65"> | When `entityType` is service: `/services/:serviceId/plugins/:pluginId`<img width="1271" alt="image" src="https://github.com/Kong/public-ui-components/assets/10095631/4e323247-3586-4f11-89b1-6f17e852f780"> |
| When `entityType` is route: `/routes/:routeId/plugins` <img width="1155" alt="image" src="https://github.com/Kong/public-ui-components/assets/10095631/82c11f09-1c55-46ca-ab53-9c701e269568"> | When `entityType` is route: `/routes/:routeId/plugins/:pluginId`<img width="1268" alt="image" src="https://github.com/Kong/public-ui-components/assets/10095631/b0953197-45ce-4f36-9217-04e7ec17265a">|
|When `entityType` is consumer: `/consumers/:consumerId/plugins`<img width="1189" alt="image" src="https://github.com/Kong/public-ui-components/assets/10095631/dcecdb29-5046-4a1a-a3bd-c342528ca553"> | When `entityType` is consumer: `/consumers/:consumerId/plugins/:pluginId`<img width="1271" alt="image" src="https://github.com/Kong/public-ui-components/assets/10095631/16ce5a92-4198-431f-8689-3dc892eaca26"> |
| When `entityType` is consumer group: `/consumer_groups/:consumerGroupId/plugins`<img width="1217" alt="image" src="https://github.com/Kong/public-ui-components/assets/10095631/debe43bf-1d91-4b14-b36a-dc685c54fa82"> | When `entityType` is consumer group: `/consumer_groups/:consumerGroupId/plugins/:pluginId`<img width="1271" alt="image" src="https://github.com/Kong/public-ui-components/assets/10095631/4dc1475c-f9ed-40c3-9cc5-47470370fef9"> |

So I tested with all 4 `entityTypes` in the sandbox, and they all work.